bucket/i2p.json

```diff
@@ -0,0 +1,56 @@
+{
+    "version": "0.9.41",
+    "description": "An anonymizing network.",
+    "homepage": "https://geti2p.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/i2p/i2p.i2p/blob/master/LICENSE.txt"
+    },
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://launchpad.net/i2p/trunk/0.9.41/%2Bdownload/i2pinstall_0.9.41_windows.exe#/dl.7z",
+    "hash": "e8d896e94696c3cb3be20cb28ea902595f82f7028622931a8ab6f451e4a3d6b5",
+    "bin": [
+        "I2P.exe",
+        "I2Psvc.exe"
+    ],
+    "shortcuts": [
+        [
+            "I2P.exe",
+            "I2P\\Start I2P (no window)",
+            "",
+            "docs\\start.ico"
+        ],
+        [
+            "I2Psvc.exe",
+            "I2P\\Start I2P (restartable)",
+            "-c \"$dir\\wrapper.config\"",
+            "docs\\start.ico"
+        ]
+    ],
+    "installer": {
+        "script": [
+            "$temp = \"$env:TEMP\\I2P\"",
+            "New-Item $temp -ItemType 'Directory' -Force | Out-Null",
+            "Move-Item \"$dir\\install.jar\" \"$temp\\install.jar\" -Force",
+            "New-Item -Path $temp -Name 'install.defaults' -Value \"INSTALL_PATH=$($dir.toString().Replace('\\', '/'))\" -Force | Out-Null",
```

Comment:
```suggestion
            "New-Item -Path $temp -Name 'install.defaults' -Value \"INSTALL_PATH=$($dir -replace '\\', '/')\" -Force | Out-Null",
```

Reply:
This doesn't work and I get
```
+ ... stall.defaults' -Value "INSTALL_PATH=$($dir -Replace '\', '/' )" -For ...
+                                            ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (\:String) []，RuntimeException
    + FullyQualifiedErrorId : InvalidRegularExpression
```

Reply:
Strange.

You could always use `\"$dir\" -replace ..` or keep this toString. It is up to you.

Reply:
I can't make -replace work... Keep toString...